google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -43,6 +43,8 @@
 
   protected abstract String getStringInternal(int columnIndex);
 
+  protected abstract String getJsonInternal(int columnIndex);
```

Comment:
```suggestion
  protected String getJsonInternal(int columnIndex) {
    throw new UnsupportedOperationException("Not implemented");
  }
```

Reply:
We should provide a default implementation not to break any users who have their own implementation of the interface.